app/server/lib/ActionHistoryImpl.ts

```diff
@@ -17,9 +17,9 @@ import {ISQLiteDB, ResultRow} from './SQLiteDB';
 // on rows and the maximum total number of bytes in the "body" column.
 // Pruning is done when the history has grown above these limits, to
 // the specified factor.
-const ACTION_HISTORY_MAX_ROWS = 1000;
-const ACTION_HISTORY_MAX_BYTES = 1000 * 1000 * 1000;  // 1 GB.
-const ACTION_HISTORY_GRACE_FACTOR = 1.25;  // allow growth to 1250 rows / 1.25 GB.
+const ACTION_HISTORY_MAX_ROWS = parseInt(process.env.ACTION_HISTORY_MAX_ROWS, 10) || 1000;
```

Comment:
It seems that the Typescript to do what you want is

```suggestion
const ACTION_HISTORY_MAX_ROWS = Number(process.env.ACTION_HISTORY_MAX_ROWS) || 1000;
```

If what you set in `env` is `NaN`, let say `"test"` it will fallback to default value 1000.